results.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+import re
+
+# Part 1
+# The first part checks how many vowels there are in each
+# string. Therefore, it checks every letter and when it finds
+# a vowels it increments the counter. 
+# The second property is found using a regex, which checks
+# if there is at least a pair of the same letter. The last property
+# is found by simply checking if there are the unwanted srings.
+
+#fone = open("output_1", "w")
+
+vowels = ["a", "e", "i", "o", "u"]
+
+firstCounter = 0
+with open("input") as in_file:
+	for row in in_file:
+		count_vowels = 0
+		for r in row:
+			if r in vowels:
+				count_vowels += 1
+
+		repLet = re.search( r'([a-z])\1', row)
+		if(repLet and count_vowels >= 3 and (("ab" not in row) and ("cd" not in row) and ("pq" not in row) and ("xy" not in row))):
+			#fone.write(row.rstrip() + "\n")
+			firstCounter += 1
+
+#fone.close()
+
+in_file.close()
+
+print("The nice strings in the first part are " + str(firstCounter))
+
+# Part 2
+# The second part splits each row in pairs 
+# and checks if exists another pair with the same
+# letters, in case they don't overlap (aaa), or 
+# they are in same position (same pair). The second 
```

Comment:
I think here you misunderstood the rule:

> It contains at least one letter which repeats with exactly one letter between them, like xyx, abcdefeghi (efe), or even aaa.

It does not mention any pair on the second rule.

Reply:
I think I've got it wrong. Probably I've got a bit confused in this part "It contains a pair of any two letters that appears at least twice in the string without overlapping, like xyxy (xy) or aabcdefgaa (aa), **but not like aaa (aa, but it overlaps).**". Therefore, I added an additional condition to check that specific case (aaa).